3rdparty/ErrorHandling/MavenException.cpp

```diff
@@ -1,6 +1,7 @@
 #include "MavenException.h"
+#include <elmavenlogger.h>
```

Comment:
@rish9511 Why not move this include statements in the header file?

Reply:
@sahil21  we should avoid including files in the header files as much as we can. 
Compilation time increases if the files are included in header files

Reply:
@rish9511 Got it.